frontend/src/views/web/challenge/my-challenge-all-submission.html

```diff
@@ -94,7 +94,7 @@ <h5 class="w-300">All Submissions</h5>
                     </thead>
                     <tbody>
                         <tr ng-repeat="key in challenge.submissionResult.results" class="result-val">
-                            <td>{{$index + 1}}</td>
+                            <td>{{$index + 1 + (challenge.currentRefPage - 1) * 100}}</td> <!--target info-->
```

Comment:
Please remove `<!--target info-->` and also add the screenshot of the interface with the changes before and after.